tasks/tasks.go

```diff
@@ -36,6 +40,21 @@ const (
 // DefaultTaskExecuter is an instance of a NewTaskExecuter.
 var DefaultTaskExecuter = NewTaskExecuter()
 
+// Map k to v in the map
```

Comment:
Technically no because these functions are unexported but I still like to have comments.